File: exercises/beecrowd/1099/1099.go
// https://judge.beecrowd.com/en/problems/view/1099
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		var x, y int
		fmt.Scan(&x, &y)

		if x > y {
			x, y = y, x
		}

		sum := 0
		for j := x + 1; j < y; j++ {
			if j%2 != 0 {
				sum += j
			}
		}
		fmt.Println(sum)
	}
}

File: exercises/beecrowd/1099/1099.ts
// https://judge.beecrowd.com/en/problems/view/1099
import * as fs from "fs";

function main(): void {
  const input = fs.readFileSync('/dev/stdin', 'utf8').trim().split('\n');
  const n = parseInt(input[0]);

  for (let i = 1; i <= n; i++) {
    let [x, y] = input[i].split(' ').map(Number);

    if (x > y) [x, y] = [y, x];

    let sum = 0;
    for (let j = x + 1; j < y; j++) {
      if (j % 2 !== 0) sum += j;
    }

    console.log(sum);
  }
}

main();
